integration_test/cases/preload.exs

```diff
@@ -552,6 +552,30 @@ defmodule Ecto.Integration.PreloadTest do
     assert p2.id == c4.post.id
   end
 
+  test "custom preload_order" do
+    p1 = TestRepo.insert!(%Post{title: "1"})
```

Comment:
```suggestion
    p = TestRepo.insert!(%Post{title: "1"})
```
We have only one `p*`, so how about using `p` or `post` instead of `p1`?

Reply:
Changed to use `post`. Thanks @Eiji7